frontend/packages/ceph-storage-plugin/src/components/block-pool/block-pool-list-page.tsx

```diff
@@ -0,0 +1,288 @@
+import * as React from 'react';
+import { TFunction } from 'i18next';
+import { useTranslation } from 'react-i18next';
+
+import { useDeepCompareMemoize, StatusIconAndText, Status } from '@console/shared';
+import { sortable, wrappable } from '@patternfly/react-table';
+import { Tooltip } from '@patternfly/react-core';
+import { referenceForModel, K8sResourceKind, referenceFor } from '@console/internal/module/k8s';
+import {
+  ListPage,
+  Table,
+  RowFunction,
+  RowFunctionArgs,
+  TableRow,
+  TableData,
+} from '@console/internal/components/factory';
+import {
+  ResourceLink,
+  ResourceKebab,
+  Kebab,
+  resourcePathFromModel,
+  humanizeBinaryBytes,
+} from '@console/internal/components/utils';
+import { usePrometheusPoll } from '@console/internal/components/graphs/prometheus-poll-hook';
+import { PrometheusEndpoint } from '@console/internal/components/graphs/helpers';
+import { StorageClassModel } from '@console/internal/models';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+
+import { menuActions } from './block-pool-menu-action';
+import { healthStateMapping } from './states';
+import { CephBlockPoolModel } from '../../models';
+import { StoragePoolKind, OcsStorageClassKind, CephClusterKind } from '../../types';
+import { CEPH_STORAGE_NAMESPACE, CEPH_EXTERNAL_CR_NAME } from '../../constants';
+import {
+  blockPoolColumnInfo,
+  isDefaultPool,
+  getScNamesUsingPool,
+  getPerPoolMetrics,
+} from '../../utils/block-pool';
+import { twelveHoursdateTimeNoYear } from '../../utils/common';
+import { PopoverHelper } from '../../utils/popover-helper';
+import { getPoolQuery, StorageDashboardQuery } from '../../queries/ceph-queries';
+import { COMPRESSION_ON } from '../../constants/storage-pool-const';
+import { cephClusterResource, scResource } from '../../resources';
+
+const getHeader = (t: TFunction) => () => {
+  return [
+    {
+      title: t(blockPoolColumnInfo.name.title),
```

Comment:
Ia doubt if this way of translating would work. We need to pass hardcoded string.
You would rather do
```suggestion
      title: blockPoolColumnInfo(t).name.title,
```

Reply:
ack